jOOL/src/main/java/org/jooq/lambda/Agg.java

```diff
@@ -82,6 +82,57 @@
         return Collectors.reducing((v1, v2) -> v2);
     }
 
+    /**
+     * Get a {@link Collector} that takes the first <code>n</code> elements from a collection.
+     * <p>The final type of this {@link Collector} is {@link Seq} type.
+     * @param n     The number of elements to be taken
+     * @param <T>   The type of element
+     * @return      A {@link Seq} of type <code>T</code>
+     */
+    public static <T> Collector<T, ?, Seq<T>> taking(long n) {
+        return Collector.of(
+                (Supplier<LinkedList<T>>) LinkedList::new,
```

Comment:
I don't think we should be using `LinkedList` here. An `ArrayList` seems to be a better choice.